packages/flutter/lib/src/material/carousel.dart

```diff
@@ -161,11 +239,14 @@ class Carousel extends StatefulWidget {
 
 class _CarouselState extends State<Carousel> {
   late double? itemExtent;
```

Comment:
Just being protective here, but `late` can be avoided if it is nullable anyway?

I'd avoid the `LateInitializationError` where we can :)